src/deviation.jl

```diff
@@ -37,10 +55,23 @@ function sqL2dist{T<:Number}(a::AbstractArray{T}, b::AbstractArray{T})
     return r
 end
 
+
 # L2 distance
+"""
+    L2dist(a, b)
+
+Compute the L2 distance between two arrays, i.e. the square root of
```

Comment:
"the square root of the squared L2 distance" is a bit of a joke. :-) It will be shorter to give the formula.